compiler/src/main/antlr/Cellmata.g4

```diff
@@ -101,7 +101,7 @@ expr : '#' # stateIndexExpr
     | PAREN_START value=expr PAREN_END # parenExpr
     | value=array_value_sized # arraySizedValueExpr
     | value=array_value_literal # arrayLiteralExpr
-    | value=expr SQ_BRACKET_START index=expr SQ_BRACKET_END # arrayLookupExpr
+    | target=expr SQ_BRACKET_START index=expr SQ_BRACKET_END # lookupExpr
```

Comment:
I like the change from `value` to `target`, it's easier to remember it as the target of the lookup operation